README.md

```diff
@@ -117,3 +117,15 @@ Refer to the [Dev Guide][devguide] for development and debugging instructions.
 [knownissues]:          https://azure.github.io/iotedge-lorawan-starterkit/dev/issues/
 [partnerinstructions]:  https://azure.github.io/iotedge-lorawan-starterkit/dev/user-guide/partner/
 [specificationsupport]: https://azure.github.io/iotedge-lorawan-starterkit/dev/#lorawan-specification-support
+
+## Create a release
+
+You can create a prerelease with the following steps:
+
+- Run the Prerelease workflow
+  Go to `Actions`, select `Prerelease` action, then select `Run workflow` and specify the release version before running the workflow.
+- Merge 2 PRs
```

Comment:
Looks duplicate with the content you have in the line below?

Reply:
These are supposed to be small titles, I changed them to be markdown heading instead to be clearer